src/Conda.jl

```diff
@@ -35,42 +35,110 @@ using Compat
 import Compat.String
 using JSON
 
-"Prefix for installation of all the packages."
-const PREFIX = abspath(dirname(@__FILE__), "..", "deps", "usr")
+const deps_file = joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")
+
+if isfile(deps_file)
+    # Includes definition for ROOTENV
+    include(deps_file)
+else
+    error("Conda is not properly configured.  Run Pkg.build(\"Conda\") before importing the Conda module.")
+end
+
+typealias Environment Union{AbstractString,Symbol}
+
+"Prefix for installation of the environment"
+function prefix(name::Symbol)
+    sname = string(name)
+    if isempty(sname)
+        throw(ArgumentError("Environment name should be non empty."))
+    end
+    return joinpath(ROOTENV, "envs", sname)
+end
+
+function prefix(path::Compat.ASCIIString)
```

Comment:
Since we are downloading Anaconda2, the path  should be ASCII paths. See conda/conda#1180.
On the other hand, a user could download Anaconda3 and use a Python2 environment by setting `CONDA_JL_HOME`.